ajax/libs/Primer/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "Primer",
+  "filename": "css/primer.css",
+  "version": "3.0.1",
+  "description": "Primer is the CSS toolkit that powers GitHub's front-end design. It's purposefully limited to common components to provide our developers with the most flexibility, and to keep GitHub uniquely *GitHubby*. It's built with SCSS and available via Bower, so it's easy to include all or part of it within your own project.",
+  "homepage": "http://primercss.io",
+  "author": "GitHub, Inc.",
```

Comment:
I think you need to modify the ```"author"``` field to mdo and jonrohan. 
[Here](http://primercss.io/about/) is the reference.